daterangepicker.js

```diff
@@ -10,10 +10,10 @@
 
   if (typeof define === 'function' && define.amd) {
     define(['moment', 'jquery', 'exports'], function(momentjs, $, exports) {
-      root.daterangepicker = factory(root, exports, momentjs, $);
```

Comment:
I can remove exports because I remove daterangepicker argument in factory function.